app/lib/params_logging.rb

```diff
@@ -0,0 +1,10 @@
+module ParamsLogging
+  IGNORE_PARAMS = %w(candidate authenticity_token).freeze
+
+  def add_params_to_request_store
+    params = request.try(:params)
```

Comment:
If this is here to guard against `request` being `nil` then you can do `request&.params`

Reply:
Being a bit paranoid, I guess, in case the module is included in a sidekiq worker or somewhere outside a web/request context. Will rename accordingly.

Reply:
On second thoughts, I'll assume request.params is available and that the module will only be included in controllers. This way I can move ```before_action :add_params_to_request_store``` to the module.